plugins/foreman_templates/4.0/index.md

```diff
@@ -12,13 +12,24 @@ You can use the native git support for simple synchronization with remote git re
 
 # 2. Installation
 
+# 2.1 Installation
```

Comment:
You have "Installation" twice here. I'd drop the 2.1 heading and allow Uninstallation to become 2.1, I think that looks better.